spam/01_spam_tutorial.py

```diff
@@ -753,11 +821,13 @@ def plot_probabilities_histogram(Y):
 
 # %% [markdown]
 # As we saw earlier, some of the data points in our `train` set received no labels from any of our LFs.
-# These examples convey no supervision signal and tend to hurt performance, so we filter them out before training.
+# These examples convey no supervision signal and tend to hurt performance, so we filter them out before training using a
+# [built-in utility](https://snorkel.readthedocs.io/en/redux/source/snorkel.labeling.html#snorkel.labeling.utils.filter_unlabeled_dataframe).
 
 # %%
 from snorkel.labeling.utils import filter_unlabeled_dataframe
```

Comment:
I'm still torn about this - it's weird that this isn't automatic or a flag in the classifier

Reply:
I agree with showing users this bit to say where the unlabeled data goes/how it's unused